<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabelleWindow</class>
 <widget class="QDialog" name="tabelleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tabelle</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Medienverwaltung</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelleWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>91</y>
     <width>821</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load_person">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Personen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Einfügen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>460</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Löschen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_media">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Medien</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_ausleihe">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Ausleihe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>0</y>
     <width>61</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Hilfe</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
